src/topbar/widget.tsx

```diff
@@ -59,7 +59,7 @@ export function TopbarElement(props: IProps): JSX.Element {
           {config.title}
         </div>
       </div>
-      <div className="specta-topbar-right">
+      {config.themeToggle && <div className="specta-topbar-right">
```

Comment:
This will hide the gear button, not only the theme dropdown. If you want to hide this button, you need to introduce a new config key